lib/list/register.js

```diff
@@ -43,28 +49,25 @@ function register () {
 	this.schema.method('getUpdateHandler', function (req, res, ops) {
 		return new UpdateHandler(list, this, req, res, ops);
 	});
-
+	/* Apply list inheritance */
 	if (this.get('inherits')) {
 		this.model = this.get('inherits').model.discriminator(this.key, this.schema);
 	} else {
 		this.model = keystone.mongoose.model(this.key, this.schema);
 	}
-	// Register the list and its field types on the Keystone instance
-	keystone.lists[this.key] = this;
-	keystone.paths[this.path] = this.key;
-	assign(keystone.fieldTypes, this.fieldTypes);
+	/* Setup search text index */
 	// If the list is configured to use a text index for search and the list doesn't explicitly define one, create (or update) one of our own
 	if (this.options.searchUsesTextIndex && !this.declaresTextIndex()) {
 		this.ensureTextIndex(function () {
 			debug('this.ensureTextIndex() done for \'' + list.key + '\'');
 		});
 	}
-	else {
-		debug('No text index need for \'' + list.key + '\' or the list defines one explicitly');
-	}
-
-	// Add a listener for model events; some of this stuff is important
-	// http://mongoosejs.com/docs/api.html#model_Model
+	/* Register the list and its field types on the Keystone instance */
+	keystone.lists[this.key] = this;
+	keystone.paths[this.path] = this.key;
+	assign(keystone.fieldTypes, this.fieldTypes);
+	/* Add listeners for model events */
+	// see http://mongoosejs.com/docs/api.html#model_Model
```

Comment:
These should be in the same comment, why are there two different ones below each other?